src/core/packages/notifications/browser-internal/src/toasts/telemetry/event_reporter.ts

```diff
@@ -20,14 +20,26 @@ interface EventPayload {
   [FieldType.TOAST_MESSAGE_TYPE]: ToastMessageType;
 }
 
-export class EventReporter {
-  private reportEvent: AnalyticsServiceStart['reportEvent'];
+export class ToastsTelemetry {
```

Comment:
nit: do you mind renaming the file to match the new class name?

Reply:
Sure, thanks for the callout